examples/analog_in/ai_temperature_signal_read.py

```diff
@@ -0,0 +1,22 @@
+﻿"""Example of analog input temperature acquisition.
+
+This example demonstrates how to acquire thermocouple measurement using
+software timing and a DAQmx device.
+"""
+
+import pprint
+
+import nidaqmx
+from nidaqmx.constants import ThermocoupleType, TemperatureUnits
+
+pp = pprint.PrettyPrinter(indent=4)
+
+
+with nidaqmx.Task() as task:
+    task.ai_channels.add_ai_thrmcpl_chan(
+        "tempTester/ai0", units=TemperatureUnits.DEG_C, thermocouple_type=ThermocoupleType.K
```

Comment:
Dev1 is good here, too. Multifunction DAQ devices can measure thermocouples (poorly :D)

Reply:
Also, don't use the devices in the testing INI. Users will not import the testing INI.

Reply:
it's replaced to "Dev1"